<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>historybox</class>
 <widget class="QWidget" name="historybox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="hisView"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>操作</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addbmBtn">
        <property name="text">
         <string>加入書籤(add bookmark)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="chooseBtn">
        <property name="text">
         <string>選擇(choose)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="helpBtn">
        <property name="text">
         <string>求助(help)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>取消(cancel)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
